packages/snap/src/veramo/accountImport.ts

```diff
@@ -101,6 +101,8 @@ export async function veramoImportMetaMaskAccount(
     if (!hederaAccountId) {
       hederaAccountId = await requestHederaAccountId(snap);
     }
+    // Attention: This line was inside of the
+    await initAccountState(snap, state, coinType, address);
```

Comment:
This is dangerous to have here. Any reason why we have this here? This will reset the entire data on the snap. @dchagastelles